### PR TITLE
Update PF Packages to 2022-2: Dark theme stylesheet

### PR DESCRIPTION
**Pull request by jcaianirh**
Update PF Packages to 2022-2

@sg00dwin this pr includes the Dark theme stylesheet release version. Also fixes for the a11y issues found in 2022-1.

### CODE REVIEW

**Comment by jcaianirh** [comment]
> @jcaianirh I'd go ahead and add the `@patternfly/patternfly/patternfly-theme-dark.css` stylesheet. Then teams can test dark mode by applying class `.pf-theme-dark` to the `html` element in developer tools (without exposing it to users yet).
> 
> cc @mattnolting

@spadgett done, and ran a quick dark theme test:

Screen Shot 2022-02-21 at 4 36 39 PM
Screen Shot 2022-02-21 at 4 35 29 PM

**Review by vikram-raj — COMMENTED** [review]
I tested this PR locally by  adding `.pf-theme-dark` class to the `html` and it enable the dark theme in the console.

**Review by sg00dwin — APPROVED** [review]
/lgtm

File: frontend/public/style.scss [hunk 1]
```diff
@@ -114,3 +114,6 @@
 @import 'components/dashboard/project-dashboard/details-card';
 
 @import 'components/nav/nav-header';
+
+// Dark theme styles from patternfly
+@import '@patternfly/patternfly/patternfly-theme-dark.css';
```
[the remark this answers is not in the document]
Comment (jcaianirh) [reply frontend/public/style.scss:119]: @rhamilto: @mattnolting and I did discuss where and how to add this, and the style.scss is the later of the two (vendor and style) to load which was preferable from a css point of view. He also suggested using the .css extension, however, I'm going to put both those questions out to him here to verify its in the correct place or not. @mattnolting would you mind verifying the location and type of the stylesheet please?

[the remark this answers is not in the document]
Comment (mattnolting) [reply frontend/public/style.scss:119]: I'm sorry, I'm must've been confused. I agree w/@rhamilto, the import just needs to be loaded **after** base patternfly imports, so doing so in `vendor.scss` is correct.

**Comment by rhamilto** [comment]
@jcaianirh, please also squash when you make the import change.

File: frontend/public/style.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (jcaianirh) [reply frontend/public/style.scss]: There is an issue upstream using vendor.scss. For now we can import the compiled css and update the import when https://github.com/openshift/console/pull/11083 is fixed.

**Comment by jcaianirh** [comment]
QE Approver:
/assign @yapei

Docs Approver:
/assign @opayne1

PX Approver:
/assign @RickJWagner

**Comment by RickJWagner** [comment]
\lgtm
\label px-approved

**Pull request merged**